src/plugins/intel_cpu/src/emitters/snippets/cpu_runtime_configurator.hpp

```diff
@@ -44,13 +44,28 @@ class CPURuntimeConfigurator : public ov::snippets::RuntimeConfigurator {
      * @param linear_ir LinearIR
      */
     void init_tensor_rank(const ov::snippets::lowered::LinearIRCPtr& linear_ir) const override;
+    void initialization(const ov::snippets::lowered::LinearIRCPtr& linear_ir) override;
     /**
      * @brief Calculate Loop parameters of Loop emitters and update these values in CPURuntimeConfig
      * @param linear_ir LinearIR
      */
     void update_loop_args(const ov::snippets::lowered::LinearIRCPtr& linear_ir) const;
 
     static const size_t rank6D;
+
+    class BrgemmCopyBLoopPortsAdjuster {
```

Comment:
>I think all "optimizers" (and other updaters) should be lowered passes which are registered in `PassPipeline` 
Yes, but we need more resources to refactor it. I don't think it makes sense to do it in this PR, since it's a temporary solution anyway.
@v-Golubev, I'll move the pass to a separate file. Also, please try to write your optimizer as pass-like as possible, so it would be easier to refactor afterwards.

Reply:
UPD: I tried to do it, but it's not possible because a friend of `CPURuntimeConfigurator` can't access protected members of it's parent (`RuntimeConfigurator`), but we need to call `get_loop_runtime_params`.
Leaving as-is, feel free to suggest an alternative.